docs/reference/gettingstartedexample/gse-debug-guide.md

```diff
@@ -0,0 +1,232 @@
+# PXIe-148X Getting Started Example - Troubleshooting Guide
+{: .no_toc }
+
+Refer to this document to get insights and suggestions on troubleshooting performance issues, error codes, and problems with using the PXIe-148x Getting Started Examples (GSE).
+
+> Note: This document references the example included with the NI-FlexRIO 22Q3 driver. Examples included in newer releases of the driver should be applicable.
+
+### Table of contents
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+---
+
+## TODO List
```

Comment:
I know we consider these working docs, but do we want to make this public with the TODO List?  There is also a TODO in the "How to debug issues with generation not starting" section.

Reply:
I think we should remove both of these.
- I skimmed csi2serdesconfig jusst now and did not see unique error codes that should be included.
- I think we should punt the 'Write section - Optimize System Settings for Streaming Performance' to a separate work item to be done later. I'm most likely to pick that up later this cycle when I have time in office to review settings on PXIe controllers.